modules/ols-0-3-6-known-issues.adoc

```diff
@@ -0,0 +1,20 @@
+// This module is used in the following assemblies:
+// * lightspeed-docs-main/release_notes/ols-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ols-0-3-6-known-issues_{context}"]
+= Known issues
+
+The following issues are identified with {ols-official} 0.3.6:
+
+* For {ocp-product-title} 4.17 and later, the {ols-short} icon disappears when you click *Create Namespace* or *Create Project* from one of the following locations in the {ocp-product-title} web console:
+
+** *Administration -> Namespaces* 
+** *Home -> Projects -> Create Project*
+** The *Project* drop-down menu located at the top of most pages
++
+Workaround: Refresh the web browser and the {ols-short} icon appears. link:https://issues.redhat.com/browse/OLS-1815[OLS-1815].
+
+* Changing the value of the `quota` parameter in the `spec.ols.quota_handlers.conf` specification of the `OLSConfig` CR file does not take effect until the currently defined quota period expires.
```

Comment:
Nit for maybe a follow up PR - spell out CR since that's the first mention in this section:
```suggestion
* Changing the value of the `quota` parameter in the `spec.ols.quota_handlers.conf` specification of the `OLSConfig` custom resource file does not take effect until the currently defined quota period expires.
```